mrobosub_perception/mrobosub_perception/timed_service.py

```diff
@@ -33,7 +34,7 @@ def __init__(self, node: rclpy.node.Node, service_name: str, ServiceType: Any, t
         self.srv = node.create_service(ServiceType, service_name, self._handle_obj_request)
 
 
-    def _handle_obj_request(self, req: Any, res: TimedResponse) -> TimedResponse:
+    def _handle_obj_request[T: TimedResponseType](self, req: Any, res: T) -> T:
```

Comment:
```suggestion
    def _handle_obj_request(self, req: Any, res: TimedResponse) -> TimedResponse:
```

We have to keep compatibility with Python 3.10 in case we need to run ROS2 Humble on the Jetson (I know I suggested this change originally, sorry)